src/common/definitions.js

```diff
@@ -311,6 +311,21 @@ const educationProgram = {
   }
 };
 
+const currentlyActiveDuty = {
+  type: 'object',
+  properties: {
+    yes: {
+      type: 'boolean'
+    },
+    onTerminalLeave: {
+      type: 'boolean'
+    },
+    nonVaAssistance: {
```

Comment:
Why is this field in here? They don't seem directly related (and they're on different pages in the UI).